tox.ini

```diff
@@ -42,10 +42,9 @@ deps =
     pytest
     pytest-cov
 commands =
-    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git
-    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/dataloader.git
-    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/models.git
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git@{posargs:main}
+    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/dataloader.git@{posargs:main}
```

Comment:
Why did you make this move here? we always end with core because the other packages will force a pull of core from the available packages on pypi and wont use the main version of the code. Is this not the behavior your seeing?

Reply:
I think that used to be the case because of the way versioneer was working and the fact that the release tags were often not on the main branch. However, with the new dev tags on the main branch this may not be an issue anymore

Reply:
I moved it out of this PR for now - we can address it elsewhere